### PR TITLE
Add new commands to enable and disable eureka and config server in consumption tier

### PR DESCRIPTION
**Pull request by CaihuaRui**
…nsumption tier

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/spring/azext_spring/_client_factory.py [hunk 1]
```diff
@@ -32,6 +32,14 @@ def cf_bindings(cli_ctx, *_):
...
 
+def cf_eureka_servers(cli_ctx, *_):
+    return cf_spring(cli_ctx).eureka_servers
+
+
+def cf_eureka_servers(cli_ctx, *_):
+    return cf_spring(cli_ctx).eureka_servers
+
+
 def cf_config_servers(cli_ctx, *_):
```
Comment (yanzhudd) [inline src/spring/azext_spring/_client_factory.py:43]: Whether is it the same as above? If so, please remove it.

File: src/spring/azext_spring/_client_factory.py [hunk 2]
```diff
@@ -32,6 +32,14 @@ def cf_bindings(cli_ctx, *_):
     return cf_spring(cli_ctx).bindings
 
 
+def cf_eureka_servers(cli_ctx, *_):
+    return cf_spring(cli_ctx).eureka_servers
+
+
+def cf_eureka_servers(cli_ctx, *_):
+    return cf_spring(cli_ctx).eureka_servers
```
Comment (yanzhudd) [inline src/spring/azext_spring/_client_factory.py:40]: Whether is it the same as above? If so, please remove it.

File: src/spring/azext_spring/custom.py [hunk 3]
```diff
@@ -21,7 +21,7 @@
 from azure.mgmt.core.tools import (parse_resource_id, is_valid_resource_id)
 from ._utils import (get_portal_uri, get_spring_sku, get_proxy_api_endpoint, BearerAuth)
 from knack.util import CLIError
-from .vendored_sdks.appplatform.v2023_03_01_preview import models, AppPlatformManagementClient
+from .vendored_sdks.appplatform.v2023_05_01_preview import models, AppPlatformManagementClient
```
Comment (yanzhudd) [inline src/spring/azext_spring/custom.py:24]: It is suggested to import specific model rather all models.

File: src/spring/azext_spring/_client_factory.py (hunk 2 above)
Comment (CaihuaRui) [reply src/spring/azext_spring/_client_factory.py]: Removed

File: src/spring/azext_spring/custom.py (hunk 3 above)
Comment (CaihuaRui) [reply src/spring/azext_spring/custom.py:24]: Should be the same version

**Comment by yanzhudd** [comment]
Please add some tests for the new added group

**Comment by CaihuaRui** [comment]
Test will be added in our CLI test repo: ASC-E2E-Test

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**